templates/citizen_project/_form.html.twig

```diff
@@ -13,37 +13,53 @@
 
 <div class="form__new" id="field-name">
     {{ form_label(form.name, 'Donnez un nom à votre projet :', {label_attr: {class: 'label__header--large'}}) }}
-    {{ form_errors(form.name) }}
-    {{ form_widget(form.name, {
-        attr: {
-            'class': 'form--full',
-            'placeholder': 'Un nom court et clair',
-        }
-    }) }}
-    <small>Ex: "Un stage pour tous"</small>
+    {% if form.name.vars.disabled %}
```

Comment:
you can use the getter `isDisabled()` of form element 
`if form.name.disabled` or `if form.name.isDisabled`

Reply:
It does not work, because it's a `FormView` class which have no method `isDisabled()`.

Reply:
oh yes, you are right 👍